packages/components/accordion/src/accordion.stories.ts

```diff
@@ -1,18 +1,20 @@
 import '@sl-design-system/button/register.js';
 import '@sl-design-system/icon/register.js';
+import { type SlToggleEvent } from '@sl-design-system/shared/events.js';
 import { type Meta, type StoryObj } from '@storybook/web-components';
-import { type TemplateResult, html } from 'lit';
+import { LitElement, type TemplateResult, html } from 'lit';
 import '../register.js';
 import { type Accordion } from './accordion.js';
 
-export type Props = Pick<Accordion, 'single'> & { items: TemplateResult };
+export type Props = Pick<Accordion, 'single'> & { items: TemplateResult; open: boolean };
```

Comment:
`open` is part of `Accordion`, so you just add it inside the `Pick`.

Reply:
<img width="975" alt="image" src="https://github.com/user-attachments/assets/0b0089f3-5ed5-4189-9389-f4d6308e9d4b">

Reply:
Ah, open is not part of Accordion, but of AccordionItem.
But I don't need this option anymore it turns out...